rollouts/Makefile

```diff
@@ -81,6 +81,22 @@ license: ## Add licenses.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
 
+.PHONY: run-in-kind
+run-in-kind: manifests kustomize ## Run the rollouts-controller-manager in a kind cluster named rollouts-management-cluster
+	kind delete cluster --name=rollouts-management-cluster
+	kind create cluster --name=rollouts-management-cluster
+	make install
+	kubectl apply -f ./manifests/crds/containercluster.yaml
+	IMG=${IMG} make docker-build
+	kind load docker-image ${IMG} --name=rollouts-management-cluster
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | sed --expression='s/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' | kubectl apply -f -
+	kubectl rollout status deployment rollouts-controller-manager --namespace rollouts-system
```

Comment:
I think we should convert this into a script.

Reply:
done!